contracts/queue/src/contract.rs

```diff
@@ -0,0 +1,202 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use snafu::ResultExt;
+
+use cosmwasm_std::{
+    from_slice, to_vec, Api, Binary, Env, Extern, Order, ParseErr, Response, Result, SerializeErr,
+    Storage,
+};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct InitMsg {}
+
+// we store one entry for each item in the queue
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct State {
```

Comment:
What about calling this just `Item`, as you need all those items to create the contract's state?

Reply:
Lazy reuse of hackatom names. I will update naming